finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,11 @@
+var frogs = new Array(10); 
```

Comment:
No need for the array here- the loop makes any console log you need.

Reply:
Updated.